_posts/2021-03-23-rust-and-tell.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Rust and Tell"
+date: "2021-03-23 19:00:00 +0200"
+categories: meetup
+location: online
+links:
+    "Meetup.com": https://www.meetup.com/Rust-Berlin/events/[needs-to-be-created]/
+---
+
+
+<center>
+    <h2><a href="https://www.youtube.com/watch?v=GuShMUou6_U&feature=youtu.be">YouTube Livestream</a></h2>
+    <h2><a href="https://matrix.to/#/!nScYCdqWQUsTkFRJMb:chat.berline.rs">Join the chat on Matrix</a></h2>
+</center>
+
+## Zoom instructions
+
+At 19:00 follow the [Zoom][] meeting invite: [Zoom Meeting](https://twilio.zoom.us/j/92809068898?pwd=VHlYRFc3Qi90WjdrNDBNMjJGU0o2QT09).
+
+[Zoom]: https://zoom.us/
+
+You will automatically be voice-muted and not be able to unmute yourself during the meetup.
+Please also disable your webcam.
+The meetup will be recorded in Zoom, everyone with enabled video will be recorded as well.
+
+The text chat inside Zoom is available, please use that for questions to the speaker. If time allows a moderator will direct them towards the speaker after the talk.
+
+During the breaks we will try out Zoom’s Breakout Rooms.
+You will be randomly assigned to a breakout room with others.
+Feel free to use this to talk to people in breaks. If you don’t want to, just mute yourself or leave the room.
+You will be brought back into the main channel before the next talk starts.
+
+## Matrix instructions
+
+See [Chat instructions](/chat/).
+
+---
+
+We are delighted 🥳 🎉 to announce that the "Rust and Tell" monthly meetup is kicking off again for 2021!
```

Comment:
Let's expand here a bit, so its actionable for people:

```suggestion
We are delighted 🥳 🎉 to announce that the "Rust and Tell" monthly meetup is kicking off again for 2021!
With new organizers, new speakers and new topics we will bring you another Rust meetup, online, livestreamed to your home.

**SPEAKERS WANTED 🙏**

Send a message to [@evanxg852000](https://twitter.com/evanxg852000) or an email to [FILL ME IN](mailto:FILL ME IN).
```

Do you have a mail address you are willing to post publicly? Otherwise we could use the rustberlin@ mail I have and I'll make sure these get to you.

Reply:
I think it's always better to use a single email `rustberlin@ mail` and forward it to the organizers.
This way people don't get confused about organizers coming in or leaving out.